solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/impl/ValueFeature.java

```diff
@@ -45,6 +46,9 @@ public ValueFeature() {}
   public void init(String name, NamedParams params, int id)
       throws FeatureException {
     super.init(name, params, id);
+    final Object paramRequired = params.get("required");
```

Comment:
Just move this param out into a constant like the value_field. Everything else looks great!